test/ControllerTest.t.sol

```diff
@@ -66,6 +66,14 @@ contract ControllerTest is ControllerHelper {
         vm.stopPrank();
     }
 
+    function testControllerFees() public {
```

Comment:
Tests are Failing! No Test written that checks new controller Logic! New controller logic == funds are transferred to the treasury when epoch is resolved! 

e2e test is missing deploying new factory, vaults, and controller, => factory market index needs to continue where the current factory is!

Reply:
where would that even make sense, if I deploy a new vault factory how can markets resume at the previous index?

Reply:
tests written, solved